decisions/Shire Bree Union.txt

```diff
@@ -0,0 +1,79 @@
+# Shire Bree Union decisions: {
+# Form Shire Bree Union
+#  }
+political_decisions = {
+	shire_bree_unification = {
+		potential = {
+			OR = {
+				tag = SHI
+				tag = BRE
+			}
+		}
+		allow = {
+			war = no
+			OR = {
+				THIS = {
+					tag = BRE
+				}

```

Comment:
Don't need this `THIS` wrapper -- no need to have the engine to extra work. Just `tag = BRE` will work